sherlock_project/resources/data.json

```
@@ -608,13 +608,15 @@
     "urlMain": "https://www.dealabs.com/",
     "username_claimed": "blue"
   },
-  "DeviantART": {
-    "errorType": "status_code",
-    "regexCheck": "^[a-zA-Z][a-zA-Z0-9_-]*$",
-    "url": "https://{}.deviantart.com",
-    "urlMain": "https://deviantart.com",
-    "username_claimed": "blue"
-  },
+  "DeviantArt": {
+  "errorType": "message",
+  "errorMsg": "Llama Not Found",
+  "regexCheck": "^[a-zA-Z][a-zA-Z0-9_-]*$",
+  "url": "https://www.deviantart.com/{}",
+  "urlMain": "https://www.deviantart.com/",
+  "username_claimed": "blue",
+  "username_unclaimed": "noonewouldeverusethis"
+},
```

Remediation checks out! Just failing due to schema noncompliance. Once this is corrected we'll be good to merge into master
```suggestion
  "DeviantArt": {
  "errorType": "message",
  "errorMsg": "Llama Not Found",
  "regexCheck": "^[a-zA-Z][a-zA-Z0-9_-]*$",
  "url": "https://www.deviantart.com/{}",
  "urlMain": "https://www.deviantart.com/",
  "username_claimed": "blue"
},
```